dask/base.py

```diff
@@ -23,6 +23,41 @@ class Base(object):
 
     def visualize(self, filename='mydask', format=None, optimize_graph=False,
                   **kwargs):
+        """
+        Render the computation of this object's task graph using graphviz.
+
+        Requires `graphviz` to be installed.
```

Comment:
On Mon, Jun 6, 2016 at 4:33 PM Jim Crist <notifications@github.com> wrote:

> In dask/base.py
> <https://github.com/dask/dask/pull/1234#discussion_r65973440>:
>
> > @@ -23,6 +23,41 @@ class Base(object):
> >
> >      def visualize(self, filename='mydask', format=None, optimize_graph=False,
> >                    **kwargs):
> > +        """
> > +        Render the computation of this object's task graph using graphviz.
> > +
> > +        Requires `graphviz` to be installed.
>
> One last nitpick: sphinx doesn't like single backticks. All places where
> you do `graphviz` can you do ``graphviz``?
>

I thought the numpy convention was single backticks for references to
variables, functions, modules, etc. and double backticks for inline
literals?


> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dask/dask/pull/1234/files/140e7d97eaca069dda90e0248a01c973457b4285#r65973440>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AASE5MCoCcSkpiJWZowompjkfvLEXwsyks5qJJIrgaJpZM4IvS13>
> .
>